code/datums/gamemode/factions/bloodcult/bloodcult_items.dm

```diff
@@ -1,16 +1,350 @@
+
+/obj/item/weapon/tome/attackby(var/obj/item/I, var/mob/user)
+	if (..())
+		return
+	if (istype(I, /obj/item/weapon/talisman))
+		if (talismans.len < MAX_TALISMAN_PER_TOME)
+			if(user.drop_item(I))
+				talismans.Add(I)
+				I.forceMove(src)
+				to_chat(user, "<span class='notice'>You slip \the [I] into \the [src].</span>")
+				if (state == TOME_OPEN)
+					usr << browse_rsc('icons/tomebg.png', "tomebg.png")
+					usr << browse(tome_text(), "window=arcanetome;size=512x375")
+		else
+			to_chat(user, "<span class='warning'>This tome cannot contain any more talismans. Use or remove some first.</span>")
+
+#undef PAGE_FOREWORD
+#undef PAGE_LORE1
+
+///////////////////////////////////////TALISMAN////////////////////////////////////////////////
+
+/obj/item/weapon/talisman
+	name = "talisman"
+	desc = "A tattered parchment. You feel a dark energy emanating from it."
+	gender = NEUTER
+	icon = 'icons/obj/cult.dmi'
+	icon_state = "talisman"
+	throwforce = 0
+	w_class = W_CLASS_TINY
+	w_type = RECYK_WOOD
+	throw_range = 1
+	throw_speed = 1
+	layer = ABOVE_DOOR_LAYER
+	pressure_resistance = 1
+	attack_verb = list("slaps")
+	autoignition_temperature = AUTOIGNITION_PAPER
+	fire_fuel = 1
+	var/obj/effect/rune/attuned_rune = null
+	var/spell_type = null
+
+/obj/item/weapon/talisman/New()
+	..()
+	pixel_x=0
+	pixel_y=0
+
+/obj/item/weapon/talisman/examine(var/mob/user)
+	..()
+	if (!spell_type)
+		to_chat(user, "<span class='info'>This one however seems pretty unremarkable.</span>")
+		return
+
+	var/datum/rune_spell/instance = spell_type
+
+	if (iscultist(user) || isobserver(user))
+		if (attuned_rune)
+			to_chat(user, "<span class='info'>This one was attuned to a <b>[initial(instance.name)]</b> rune.</span>")
+		else
+			to_chat(user, "<span class='info'>This one was imbued with a <b>[initial(instance.name)]</b> rune.</span>")
+	else
+		to_chat(user, "<span class='info'>This one was some arcane drawings on it. You cannot read them.</span>")
+
+/obj/item/weapon/talisman/attack_self(var/mob/living/user)
+	if (iscultist(user))
+		trigger(user)
+
+/obj/item/weapon/talisman/proc/trigger(var/mob/user)
+	if (!user) return
+
+	if (!spell_type)
+		if (src != user.get_active_hand())
+			user.put_in_hands(src)//removing from a tome
+		return
+
+	if (attuned_rune)
+		attuned_rune.trigger(user)
+	else
+		new spell_type(user, src)
+
+	if (istype(loc,/obj/item/weapon/tome))
+		var/obj/item/weapon/tome/T = loc
+		T.talismans.Remove(src)
+	qdel(src)
+
+/obj/item/weapon/talisman/proc/imbue(var/mob/user, var/obj/effect/rune/R)
+	if (!user || !R) return
+
+	var/datum/rune_spell/spell = get_rune_spell(user,null,"examine",R.word1, R.word2, R.word3)
+	if(initial(spell.talisman_absorb) == RUNE_CANNOT)
+		user.drop_item(src)
+		src.forceMove(get_turf(R))
+		R.attack_hand(user)
+	else
+		if (attuned_rune)
+			to_chat(user, "<span class='warning'>This [src] is already linked to a rune.</span>")
```

Comment:
> This the talisman is already imbued

Reply:
so \This works here?

Reply:
you just made that up on the spot
you need to use `This [name]`

Reply:
oh well duh.